onnxruntime/core/mlas/lib/sgemm.cpp

```diff
@@ -1380,77 +1407,97 @@ Return Value:
 
 --*/
 {
-    float PanelA[MLAS_SGEMM_TRANSA_ROWS * MLAS_SGEMM_PACKED_STRIDEK];
-
-    //
-    // Step through each slice of matrix B along the N dimension.
-    //
-
-    size_t CountN;
+#if !(defined(USE_KLEIDIAI) && !defined(_MSVC_LANG))
+    MLAS_UNREFERENCED_PARAMETER(TransB);
+#endif
 
-    for (size_t n = 0; n < RangeCountN; n += CountN) {
+#if defined(USE_KLEIDIAI) && !defined(_MSVC_LANG)
+    if (UseKleidiAISgemm(TransA, TransB, AlignedN, K)) {
+        const size_t dst_stride = ldc * sizeof(float);
 
-        const size_t SliceStartN = RangeStartN + n;
+        const size_t rhs_packed_offset =
+            kai_get_rhs_packed_offset_matmul_clamp_f32_f32p2vlx1_f32p2vlx1biasf32_sme2_mopa(RangeStartN, K);
+        const void* rhs_ptr = reinterpret_cast<const void*>(
+                reinterpret_cast<const char *>(PackedB) + rhs_packed_offset);
 
-        CountN = std::min(RangeCountN - n, size_t(MLAS_SGEMM_PACKED_STRIDEN));
+        kai_run_matmul_clamp_f32_f32p2vlx1_f32p2vlx1biasf32_sme2_mopa(M, RangeCountN, K,
+            A, rhs_ptr, C, dst_stride, sizeof(float),
+            -std::numeric_limits<float>::max(), std::numeric_limits<float>::max());
```

Comment:
Very minor ignorable nit: Maybe use std::numeric_limits<float>::lowest() instead of -std::numeric_limits<float>::max() ?